writing-specifications.rst

```diff
@@ -244,15 +263,13 @@ A step can have the following types of parameters:
 * Table parameters
 * Special parameters
 
-
-1. Simple parameters
+Simple parameters
 ^^^^^^^^^^^^^^^^^^^^
 
 Simple parameters are values, which are used in a step within double quotes.
 
 Example
 ^^^^^^^
-
 In the following examples, ``Die Hard``, ``gauge-java``, and ``100`` are values within double quotes and are used in steps: 
```

Comment:
Change `Die Hard` to `Cup Cakes` Please make sure all occurrences of `Die Hard` is changed to `Cup Cakes`